src/promptflow/promptflow/_cli/_pf/_connection.py

```diff
@@ -178,24 +177,24 @@ def create_connection(file_path, params_override=None, name=None):
     if name:
         params_override.append({"name": name})
     connection = load_connection(source=file_path, params_override=params_override)
-    existing_connection = _client.connections.get(connection.name, raise_error=False)
+    existing_connection = PFClient.get_instance().connections.get(connection.name, raise_error=False)
```

Comment:
no need to change like this

Reply:
done, I modified it to add a singleton function to the current file,